test/linalg3.jl

```diff
@@ -161,7 +161,10 @@ for elty in (Float32, Float64, Complex64, Complex128)
                                               0  -0.000000000000002   3.000000000000000])
 end
 
+OLD_STDERR = STDERR
+redirect_stderr(open("/dev/null", "w"))
```

Comment:
@yuyichao Thank you for your help, I did as you suggested. I check the content of the warning as well.